source/brailleTables.py

```diff
@@ -1,16 +1,26 @@
 # A part of NonVisual Desktop Access (NVDA)
 # This file is covered by the GNU General Public License.
 # See the file COPYING for more details.
-# Copyright (C) 2008-2019 NV Access Limited, Joseph Lee, Babbage B.V.
+# Copyright (C) 2008-2020 NV Access Limited, Joseph Lee, Babbage B.V., Julien Cochuyt
 
 """Manages information about available braille translation tables.
 """
 
 import collections
+from configobj import ConfigObj
+from io import StringIO
+from typing import BinaryIO, Generator
+
 
 #: The directory in which liblouis braille tables are located.
 TABLES_DIR = r"louis\tables"
 
+#: Table to use for both input and output if configuration is invalid.
+FALLBACK_TABLE_NAME = "en-ueb-g1.ctb"

```

Comment:
Rather than duplicate the definition for the default, you could look up the default value from the config spec: `config.conf.getConfigValidation(("braille", "cursorBlinkRate")).kwargs["max"]`

Reply:
Makes sense. Thanks for pointing it out.

Reply:
Done: e25872c
Thanks for teaching me how to retrieve these default values.